<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputStrDialog</class>
 <widget class="QWidget" name="InputStrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Streams</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="10">
    <widget class="QLabel" name="Label7">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="4">
    <widget class="QDoubleSpinBox" name="sBNmeaPosition1">
     <property name="decimals">
      <number>9</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="12">
    <widget class="QLineEdit" name="lEFilePath3">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="12">
    <widget class="QLineEdit" name="lEFilePath1">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="13">
    <widget class="QPushButton" name="btnFile1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="13">
    <widget class="QPushButton" name="btnReceiverOptions3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QComboBox" name="cBNmeaReqL">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>OFF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Latitude/Longitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single Solution</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="13">
    <widget class="QPushButton" name="btnReceiverOptions1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QComboBox" name="cBFormat3">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="8" colspan="2">
    <widget class="QDoubleSpinBox" name="sBNmeaPosition2">
     <property name="decimals">
      <number>9</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QComboBox" name="cBFormat2">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="10">
    <widget class="QDoubleSpinBox" name="sBNmeaPosition3">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>8000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QComboBox" name="cBFormat1">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="3">
    <widget class="QComboBox" name="cBStream1">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="4" colspan="3">
    <widget class="QComboBox" name="cBStream2">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="btnStream2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="13">
    <widget class="QPushButton" name="btnPosition">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="cBStreamC1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>(1) Rover</string>
     </property>
    </widget>
   </item>
   <item row="13" column="11" colspan="3">
    <widget class="QPushButton" name="btnCancel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <widget class="QLabel" name="lblF2">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QPushButton" name="btnReceiverOptions2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="2">
    <widget class="QPushButton" name="btnCmd2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QComboBox" name="cBTimeSpeedL">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x0.2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x10</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLabel" name="lblF3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QLabel" name="lblNmea">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Transmit NMEA GGA to Base Station</string>
     </property>
    </widget>
   </item>
   <item row="13" column="6" colspan="5">
    <widget class="QPushButton" name="btnOk">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="13">
    <widget class="QPushButton" name="btnFile2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="12">
    <widget class="QLineEdit" name="lEFilePath2">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="Label10">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Input Stream</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QCheckBox" name="cBStreamC3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>(3) Correction</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLineEdit" name="lETimeStartE">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8" colspan="2">
    <widget class="QLabel" name="Label11">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Cmd</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QLabel" name="Label1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Opt</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="lblF1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Input File Paths</string>
     </property>
    </widget>
   </item>
   <item row="11" column="13">
    <widget class="QPushButton" name="btnFile3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QCheckBox" name="cBTimeTagC">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="Label5">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="Label6">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Opt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="btnStream1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="2">
    <widget class="QPushButton" name="btnCmd1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="btnStream3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8" colspan="2">
    <widget class="QPushButton" name="btnCmd3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCheckBox" name="cBStreamC2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>(2) Base Station</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="3">
    <widget class="QComboBox" name="cBStream3">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FTP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HTTP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblResetCmd">
     <property name="text">
      <string>Reset Cmd</string>
     </property>
    </widget>
   </item>
   <item row="7" column="10">
    <widget class="QLabel" name="lblMaxBaseLine">
     <property name="text">
      <string>MaxBaseline</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="9">
    <widget class="QLineEdit" name="lEResetCmd"/>
   </item>
   <item row="13" column="5">
    <widget class="QCheckBox" name="cB64Bit">
     <property name="text">
      <string>64bit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="13">
    <widget class="QDoubleSpinBox" name="sBMaxBaseLine">
     <property name="suffix">
      <string> km</string>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBStreamC1</tabstop>
  <tabstop>cBStream1</tabstop>
  <tabstop>btnStream1</tabstop>
  <tabstop>btnCmd1</tabstop>
  <tabstop>cBFormat1</tabstop>
  <tabstop>btnReceiverOptions1</tabstop>
  <tabstop>cBStreamC2</tabstop>
  <tabstop>cBStream2</tabstop>
  <tabstop>btnStream2</tabstop>
  <tabstop>btnCmd2</tabstop>
  <tabstop>cBFormat2</tabstop>
  <tabstop>btnReceiverOptions2</tabstop>
  <tabstop>cBStreamC3</tabstop>
  <tabstop>cBStream3</tabstop>
  <tabstop>btnStream3</tabstop>
  <tabstop>btnCmd3</tabstop>
  <tabstop>cBFormat3</tabstop>
  <tabstop>btnReceiverOptions3</tabstop>
  <tabstop>cBNmeaReqL</tabstop>
  <tabstop>sBNmeaPosition1</tabstop>
  <tabstop>sBNmeaPosition2</tabstop>
  <tabstop>sBNmeaPosition3</tabstop>
  <tabstop>btnPosition</tabstop>
  <tabstop>lEFilePath1</tabstop>
  <tabstop>btnFile1</tabstop>
  <tabstop>lEFilePath2</tabstop>
  <tabstop>btnFile2</tabstop>
  <tabstop>lEFilePath3</tabstop>
  <tabstop>btnFile3</tabstop>
  <tabstop>cBTimeTagC</tabstop>
  <tabstop>cBTimeSpeedL</tabstop>
  <tabstop>lETimeStartE</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
